content/departments/technical-success/support/process/Support Agent and Customer Slack Channels

```diff
@@ -0,0 +1,13 @@
+##Support Agent and Customer Slack Channels 
+
+To better support our customers we use a Support Agent that will need to be integrated into customer slack channels. This will allow us to receive customer request quickly and directly triage the request from Slack. To ensure that the Support Agent is enabled there are a few steps that need to be completed. 
```

Comment:
I think this can be cut down and also clarify that this is needed for slack support. How about:

We currently offer Support via Slack Channels for Enterprise customers through a Support Agent integration that _must_ be integrated into the Slack Channel. This integration is a requirement to ensure we can provide a consistent support experience to our customers in accordance with our Service Level Agreements (SLAs). To enable the Support agent, the following steps need to be completed.